packages/unchained-client/src/ethereum/parser/__tests__/ethereum.test.ts

```diff
@@ -977,7 +977,10 @@ describe('parseTx', () => {
         address,
         chainId: 'eip155:1',
         confirmations: txMempool.confirmations,
-        data: undefined,
+        data: {
+          method: 'stake',
```

Comment:
Is the comment above done now?

`// TODO: parse pending LP Token send to staking contract using stake() contract call`.